translations/monero-core_fr.ts

```diff
@@ -1107,42 +1113,42 @@ The old wallet cache file will be renamed and can be restored later.
     <message>
         <location filename="../pages/Settings.qml" line="160"/>
         <source>Successfully rescanned spent outputs.</source>
-        <translation type="unfinished"></translation>
+        <translation>Rescanné vos sorties déponsé avec succès</translation>
```

Comment:
Transactions sortantes rescannées avec succès

Reply:
I personally preffer @glv2 's suggestion of "Sorties dépensées rescannées avec succès." as "spent outputs" isn't really the same as "outgoing transactions"